packages/component-elife-ui/client/molecules/ModalOverlay.js

```diff
@@ -52,6 +52,18 @@ class ModalOverlay extends React.Component {
     this.overlay = React.createRef()
   }
 
+  componentDidUpdate(prevProps, prevState, snapshot) {
```

Comment:
Is this also in the Pubsweet modal component? If not, might be worth putting in a MR / raising an issue there too so this doesn't reappear when we upgrade and start using the shared component